_i18n/de/resources/moneropedia/bulletproofs.md

```diff
@@ -1,31 +1,31 @@
 ---
-terms: ["bulletproofs", "bulletproof"]
-summary: "a new kind of range proofs replacing RingCT in transactions to obfuscate the amounts sent"
+terms: ["bulletproofs", "bulletproof", "Bulletproof", "Bulletproofs"]
+summary: "Eine neue Art von 'Range-Proofs', welche Ring-CTs in Transaktionen ersetzen, um die gesendeten Beträge zu verschleiern"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
-@RingCT was introduced to obfuscate transaction amounts. One goal of @RingCT was to prove the sum of inputs - outputs in the @transaction was equal to 0, and all outputs were positive numbers.  
-To accomplish this, two kind of ring signatures were constructed: One ring signature for the whole transaction (to prove the sum is 0), and a set of ring signatures for the subsets of transaction bits (to prove the outputs are positive numbers), then combined together using originally Schnorr signatures (and later replaced by Borromean ring signature).  
-While it was doing the job, a big drawback was the huge size of such a ringCT transaction.
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
+@Ring-CTs wurden und werden eingesetzt, um Transaktionsbeträge zu verschleiern. Ein Ziel dieser @Ring-CTs ist es, nachzuweisen, dass der Endbetrag von Inputs zu Outputs gleich null ist, und alle Outputs positive Zahlen sind.
+Um dies zu erreichen, werden zwei Arten von Ringsignaturen konstruiert: eine Ringsignatur für die gesamte Transaktion (um nachzuweisen, dass die Endsumme null ergibt) und eine Reihe von Ringsignaturen für die Teilmengen von Transaktionsstücken (um nachzuweisen, dass die Outputs positive Zahlen sind). Diese werden anschließend kombiniert (zu Beginn passierte das durch Schnorr-Signaturen, welche später von Borromäischen Ringsignaturen abgelöst wurden). Während dies grundsätzlich funktioniert, gibt es einen großen Haken an der Sache: die immense Größe einer solchen Ring-CT-Transaktion.
```

Comment:
```suggestion
Um dies zu erreichen, werden zwei Arten von Ringsignaturen konstruiert: eine Ringsignatur für die gesamte Transaktion (um nachzuweisen, dass die Endsumme null ergibt) und eine Reihe von Ringsignaturen für die Teilmengen von Transaktionsstücken (um nachzuweisen, dass die Outputs positive Zahlen sind). Diese werden anschließend kombiniert (zu Beginn passierte das durch Schnorr-Signaturen, welche später von Borromäischen Ringsignaturen abgelöst wurden). Während dies grundsätzlich funktioniert, gibt es einen großen Haken an der Sache: die immense Größe einer solchen RingCT-Transaktion.
```

Reply:
During the whole translation I wrote Ring-CT with the hyphen, I would prefer to keep it like that since it technically doesn't change anything..?